src/App.js

```diff
@@ -4,10 +4,11 @@ import {createDrawerNavigator} from '@react-navigation/drawer';
 import {NavigationContainer} from '@react-navigation/native';
 
 import LessonBase from './screens/lessonBase';
+import LevelOverview from './screens/levelOverview';
 
 const HomeScreen = ({navigation}) => (
   <View style={{flex: 1, alignItems: 'center', justifyContent: 'center'}}>
-    <Button onPress={() => navigation.navigate('Level One')} title='Go to Level One' />
+    <LevelOverview />
```

Comment:
actually, we wanna keep the home page, but replace level1-4 by the level overview page. you see it when you go to a level, and each level overview takes you to the lesson of that level